sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/BlockBlobItem.java

```diff
@@ -29,7 +29,32 @@ public class BlockBlobItem {
      * @param contentMd5 Content MD5 of the block blob.
      * @param isServerEncrypted Flag indicating if the block blob is encrypted on the server.
      * @param encryptionKeySha256 The encryption key used to encrypt the block blob.
+     * @param encryptionScope The encryption scope used to encrypt the block blob.
+     * @param versionId The version identifier of the block blob.
+     */
+    public BlockBlobItem(final String eTag, final OffsetDateTime lastModified, final byte[] contentMd5,
```

Comment:
adding this below the constructors instead of above will maintain the chronological progression of these constructors, making it easier to decipher later.